samples/js/chat_sample/README.md

```diff
@@ -0,0 +1,48 @@
+# JavaScript chat_sample that supports most popular models like LLaMA 3
+
+This example showcases inference of text-generation Large Language Models (LLMs): `chatglm`, `LLaMA`, `Qwen` and other models with the same signature. The application doesn't have many configuration options to encourage the reader to explore and modify the source code. For example, change the device for inference to GPU. The sample fearures `Pipeline.LLMPipeline` and configures it for the chat scenario.
+
+## Download and convert the model and tokenizers
+
+To convert model you have to use python package `optimum-intel`.
+The `--upgrade-strategy eager` option is needed to ensure `optimum-intel` is upgraded to the latest version.
+
+Install [../../export-requirements.txt](../../export-requirements.txt) to convert a model.
+
+```sh
+pip install --upgrade-strategy eager -r ../../export-requirements.txt
+optimum-cli export openvino --trust-remote-code --model TinyLlama/TinyLlama-1.1B-Chat-v1.0 TinyLlama-1.1B-Chat-v1.0
+```
+
+## Run:
+
+Compile GenAI JavaScript bindings archive first using the instructions in [../../../src/js/README.md](../../../src/js/README.md#build-bindings).
+
+Run `npm install` in current folder and then run the sample:
```

Comment:
The instruction to run `npm install` in `samples/js/chat_sample` here contradicts to the instructions in `src/js/README.md`:
> To use this package locally use `npm link` in `src/js/` directory and `npm link genai-node` in the folder where you want to add this package as a dependency

Please clarify the instructions

Reply:
Don't see contradiction, because in the repository, the structure defines and we can easily refer to another project directory to install dependency from it.
Way with `npm link` is extra complexity for genai samples, but it still actual way to use genai npm package in the case when user wants to use it in another local nodejs project.

Reply:
Either align the instructions or explain that difference in readme.

Reply:
What is the goal of this change?
There are two different scenarios: 
1. Specification of relative path to dependency in the single project (current repo)
2. User wants to use local build in another local project

Each readme contains information about each usecase specifically. 

Also, later instruction will be changed anyway, because instead of containing information about local build, project README.md will contain info about installation from npm registry.